lib/gpu/lib/src/buffer.dart

```diff
@@ -0,0 +1,73 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+import 'dart:nativewrappers';
+import 'dart:typed_data';
+
+import 'context.dart';
+
+/// A buffer data range.
+class BufferView {
+  /// The buffer of this view.
+  final Buffer buffer;
+  /// The start of the view, in bytes starting from the beginning of the
+  /// [buffer].
+  final int offsetInBytes;
+  /// The end of the view
+  final int lengthInBytes;
+
+  /// Create a new view into a buffer on the GPU.
+  const BufferView(
+      {required this.buffer,
+      required this.offsetInBytes,
+      required this.lengthInBytes});
+}
+
+/// A buffer that can be referenced by commands on the GPU.
+mixin Buffer {}
+
+/// [HostBuffer] is a [Buffer] which is allocated on the host and lazily
```

Comment:
I would clarify what "host" means in this context, specifically: CPU resident memory native heap, not GPU memory or Dart heap.

Reply:
Done.